src/App.tsx

```diff
@@ -15,36 +16,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  DEFAULT = 'DEFAULT',
+  ALPHABETICAL = 'ALPHABETICAL',
+  LENGTH = 'LENGTH',
+  REVERSE = 'REVERSE',
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
```

Comment:
Not an error - but in most cases you can skip typing on `useState`

```suggestion
  const [sortField, setSortField] = useState(SortType.DEFAULT);
```